bueno/cli.rs

```diff
@@ -112,6 +118,25 @@ pub fn parse_cli() -> ExitCode {
                 code = ExitCode::FAILURE;
             }
         }
+        Some(("lint", sub_matches)) => {
+            let default_glob = "**/*".to_string();
+            let glob = sub_matches
+                .get_one::<String>("glob")
+                .unwrap_or(&default_glob);
+
+            let result = lint(LintOptions { glob });
+
+            match result {
+                Ok(false) => {
```

Comment:
and here we'd match against `Errors(errors)` and use `print_lint_errors` to printeprintln them